k2/csrc/ragged_ops.h

```diff
@@ -82,6 +81,26 @@ void LogSumPerSublist(Ragged<T> &src, T initial_value, Array1<T> *dst_values) {
   ApplyOpPerSublist<T, LogAdd<T>>(src, initial_value, dst_values);
 }
 
+
+/*
+  Output to an array `max_values` the arg-max within each sub-list along the
+  last axis of `src` i.e. the max taken over the last axis), i.e. the index
+  within `src.values` of the maximum element of that sub-list, or -1
+  if the sub-list was empty or had value <= `initial_value`.  [or maybe
+  < `initial_value`, must check, that may be undefined.]
+
+     @param [in] src            Input ragged array; must have src.NumAxes()
+                                >= 2. src.values is allowed to be empty.
+     @param [out] argmax        Array to which the argmax indexes will be written.
+                                max_values->Dim() == src.TotSize(src.NumAxes() - 1).
```

Comment:
Should this be
```cpp
argmax->Dim() == src.TotSize(src.NumAxes() - 2)
```
?

Reply:
Sure, seems it's a historical issue, also updated the documentation of other RaggedOps